operator/src/dealManager.js

```diff
@@ -78,7 +78,7 @@ class DealManager {
     async verifyDepositAmountAsync(sender, amount) {
         debug('Verifying balance for deposit', sender, amount);
         const account = this.web3.utils.toChecksumAddress(sender);
-        const balance = await this.contract.methods.getParticipantBalance(account).call({from: this.scClient.getOperatorAccount()});
+        const balance = await this.contract.methods.getParticipantBalance(account).call();
```

Comment:
For the reviewer:
`this.contract` is created from `this.web3` which has the `defaultAccount` set in the server startup.
same story below.

Reply:
correction:
this.contract sets "from" based on "defaultAccount"